templates/demo-store/app/routes/($locale).collections.$collectionHandle.tsx

```diff
@@ -94,9 +110,8 @@ export async function loader({params, request, context}: LoaderArgs) {
     collections: CollectionConnection;
```

Comment:
Now that we have typesgen, we should not need to pass this explicit type to the storefront.query